src/mb/mizinkobusters/elysium/command/Xpranking.java

```diff
@@ -0,0 +1,66 @@
+package mb.mizinkobusters.elysium.command;
+
+import org.bukkit.Bukkit;
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandExecutor;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+
+import java.util.*;
+
+public class Xpranking implements CommandExecutor {
+
+    @Override
+    public boolean onCommand(CommandSender sender, Command cmd, String label, String[] args) {
+
+        if (!cmd.getName().equalsIgnoreCase("xpranking")) {
+            return true;
+        }
+
+        Player player;
+        if (!(sender instanceof Player)) {
+
+            sender.sendMessage("§cOnly players can execute this command.");
+            return true;
+
+        } else {
+            player = (Player) sender;
+        }

```

Comment:
変数の初期化はあまり離れない方がいいので（好みの問題ですが基本3行以内)下記の方がよいです
```suggestion
        if (!(sender instanceof Player)) {
            sender.sendMessage("§cOnly players can execute this command.");
            return true;
        } 
        
        Player player = (Player) sender;
```